pkg/pool-weighted/test/foundry/LiquidityApproximationWeighted.t.sol

```diff
@@ -71,47 +72,84 @@ contract LiquidityApproximationWeightedTest is LiquidityApproximationTest {
 
     // Tests varying weight
 
+    // #region addLiquidityUnbalanced
```

Comment:
These region tags were removed in another PR; shouldn't reintroduce here. Here and elsewhere.